backend/src/index.ts

```diff
@@ -1,36 +1,65 @@
-import FireFly, { FireFlySubscriptionBase } from "@hyperledger/firefly-sdk";
-import express from "express";
+import FireFly from "@hyperledger/firefly-sdk";
 import bodyparser from "body-parser";
-import simplestorage from "../contracts/simple_storage.json";
-import { v4 as uuidv4 } from "uuid";
+import express from "express";
+import simplestorage from "../../solidity/artifacts/contracts/simple_storage.sol/SimpleStorage.json";
+import token from "../../solidity/artifacts/contracts/token.sol/Token.json";
 
 const PORT = 4001;
 const HOST = "http://localhost:5000";
 const NAMESPACE = "default";
+const SIMPLE_STORAGE_ADDRESS = "ContractAddressHere";
```

Comment:
Should we add something to the README about ensuring they fill in this variable? or we could pull from an environment variable instead.

Reply:
When the contracts compile, the console instructs them where to put the addresses. I think the candidate should be the one to productize and move it to an env variable.